src/oge/output_data.py

```diff
@@ -411,33 +385,29 @@ def write_generated_averages(
         skip_outputs (bool): whether to save data or not.
     """
     if not skip_outputs:
-        fuel_type_production = (
-            ba_fuel_data.groupby(["fuel_category"]).sum(numeric_only=True).reset_index()
+        national_avg = (
+            ba_fuel_data.groupby(["fuel_category"])[DATA_COLUMNS]
+            .sum(numeric_only=True)
```

Comment:
Should we use `mean` to get the average for each fuel category?

Reply:
No, here, we need to get total emissions and total mwh before dividing them to get the rates.